test/notifications_test.js

```diff
@@ -81,29 +85,40 @@ describe('notifications', function() {
 
   serverManager.inject();
 
-  beforeEach(function*() {
-    mq.connect(mqSocket);
-    yield chakram.post(subscriptionsUrl, subscription);
-  });
+  it('should mark reminder as errored if no subscription exists', function*() {
+    let hasOneMessageBeenSent = false;
+    waitForMqMessage().then(() => {
```

Comment:
That's right. I don't wont to block the test execution, waiting for a timeout on `waitForMqMessage()`. That's why I used regular promises